packages/utilities/psammead-storybook-helpers/src/text-variants.js

```diff
@@ -7,6 +7,9 @@ const TEXT_VARIANTS = {
     locale: 'om',
     timezone: 'Africa/Addis_Ababa',
     articlePath: '/afaanoromoo/articles/c4g19kgl85ko',
+    brandBackgroundColour: '#B80000',
```

Comment:
As these values seem to be mostly the same throughout this file shall we define them as a default const up top and use that default const instead specifying the same colours for every service? 

This will make it quicker and easier if we ever need to update the values

Reply:
Yep - how do I do that :P

Reply:
At the top of the file I define a few more consts like so

```
const DEFAULT_BRAND_BACKGROUND_COLOUR = '#B80000'
```

Then use those new constants throughout the file replacing the hardcoded values for each service like so

```
{
articlePath: '/afaanoromoo/articles/c4g19kgl85ko',
brandBackgroundColour: DEFAULT_BRAND_BACKGROUND_COLOUR,
brandForegroundColour: DEFAULT_BRAND_FORGROUND_COLOUR,
brandBorderColour: DEFAULT_BRAND_BORDER_COLOUR,
}
```

Reply:
Okay cool, thanks

Reply:
Done :)

Reply:
Ace, this will make it easier to maintain. Thanks.